README.md

```diff
@@ -28,7 +28,7 @@
   </a>
 </p>
 
-> _*Warning*_: Currently this project has an issue being tracked [here](https://github.com/ngrok/kubernetes-ingress-controller/issues/208) which can cause the controller to provide ingress to a service even if it failed to configure an authentication module like Oauth on a particular route due to a configuration or intermittent error. Additionally, even when there aren't errors, the controller first sets up ingress and then applies route modules like Oauth immediately after, but there is a brief period where ingress without authentication is provided. This issue is being tracked [here](https://github.com/ngrok/kubernetes-ingress-controller/issues/219). Until this issue is resolved, it's not recommended to use this with security sensitive applications.
+> _*Warning*_: The controller first sets up ingress and then applies route modules like Oauth immediately after, but there is a brief period where ingress without authentication is provided. This issue is being tracked [here](https://github.com/ngrok/kubernetes-ingress-controller/issues/219). Until this issue is resolved, it's not recommended to use this with security sensitive applications.
```

Comment:
The new message doesn't make it clear that there are still terminal cases where the controller will, even with correct authentication modules configured, leave your services exposed to the internet.
Namely, unless I'm reading this PR incorrectly, valid updates can still leave you in terminal insecure states.

I think the new wording, with "brief period", is misleading since it's easily possible to also end up with non-brief periods of insecurity.

Reply:
This message is to be changed and moved soon, let's keep that out of scope for this PR.